my_robot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  #Position Controllers
  base_lidar_to_motor_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: "lidar_base_to_motor_joint"
    pid: {p: 1.0, i: 1.0, d: 0.0}